src/gluonts/dataset/split/splitter.py

```diff
@@ -77,8 +77,8 @@ def from_data_entry(
         if freq is None:
             freq = item["start"].freq
 
-        index = pd.period_range(
-            start=item["start"], freq=freq, periods=len(item["target"])
+        index = pd.date_range(
+            start=item["start"].to_timestamp(), freq=freq, periods=len(item["target"])
```

Comment:
Also this one needs not be changed. You probably started this branch from before #1980 was merged?